src/core/execute.c

```diff
@@ -453,6 +453,12 @@ int exec_spawn(
          * handoff timestamp. */
         dual_timestamp_now(&start_timestamp);
 
+        r = 0;
+        if (unit->manager->starting_ambient_set != 0 && getpid_cached() != 1)
```

Comment:
MANAGER_IS_USER() instead of checking pid?

Reply:
DONE.